packages/cli/src/commands/setup/ui/libraries/tailwindcss.js

```diff
@@ -223,10 +223,12 @@ export const handler = async ({ force, install }) => {
         const prettierConfigPath = path.join(rwPaths.base, 'prettier.config.js')
         // Add tailwindcss ordering plugin to prettier
         const prettierConfig = fs.readFileSync(prettierConfigPath, 'utf-8')
-        const tailwindConfigPath = path.relative(
-          rwPaths.base,
-          path.join(rwPaths.web.config, 'tailwind.config.js')
-        )
+        const tailwindConfigPath = path
+          .relative(
+            rwPaths.base,
+            path.posix.join(rwPaths.web.config, 'tailwind.config.js')
+          )
+          .replace(/\\/g, path.posix.sep)
```

Comment:
I actually prefer `'/'` here. Yes, I read Peter's comment. But I still prefer `'/'` 🙂 

Also I think we can use `replaceAll` here instead of the regex.

Reply:
@Tobbe IDK what is next to do on this, so I am going to sit back for now. Please let me know if there is something else I need to do.

Reply:
I ran the changes on my windows machine, and now I get Green checkmarks and no errors. 
 ``` PS C:\Users\Kirby\Desktop\redwood\test-project> yarn rw g component test
  √ Generating component files...
    √ Successfully wrote file `./web\src\components\Test\Test.test.js`
    √ Successfully wrote file `./web\src\components\Test\Test.stories.js`
    √ Successfully wrote file `./web\src\components\Test\Test.js` ```